crypto/string.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package crypto
+
+import (
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type String string
+
+const bsonKey = "aes-256"
```

Comment:
This key does not include information about the cipher mode (Cipher Feedback `CFB`).
```suggestion
const bsonKey = "AES_256_CFB"
```